PoTv2Debug/PoTv2Debug.ino

```diff
@@ -116,10 +119,10 @@ void loop()
   state.UpdateRotEncSwitch();
 
   // handle rotary encoder state
-  if (state.UpdateRotEnc(RotaryEncoder.read()))
-  {
-    RotaryEncoder.write(state.GetRotEncValue());
-  }
+  rotEncRetval = state.ProcessRotEnc(RotaryEncoder.read()); 
```

Comment:
The root of the problem (thus, the root of the fix) was that I'd not familiarized myself enough with the actual mechanics of the Encoder library. This new method appears to be rock solid, and can be implemented on the actual prototype software.